### ISSUE TITLE
question about SDN fine-tune on KITTI

### ISSUE THREAD

**Issue by linbaiwpi** [mileyan/pseudo-LiDAR_e2e#12]
All the codes mentioned below are from **PointRCNN** folder.

I noticed in 'kitti_dataset.py', dataloader reads depth ground truth from "KITTI/training/depth_map" folder. If I understand correctly, we can generate this folder by using the following command from **Pseudo_Lidar_V2** repo.
`python ./src/preprocess/generate_depth_map.py --data_path path-to-KITTI/ --split_file ./split/trainval.txt`

I did so and displayed the file named "001596.npy". The result gives a sparse depth map (shown below). And I think this sparse depth map is used as ground truth to calculate the "Depth loss" in Figure 3.
![1693](https://user-images.githubusercontent.com/31357378/97602743-0b6e1880-19e2-11eb-8c4a-8f810de2c39a.PNG)

In _Section 4.2 Depth Estimation_, you mentioned the pre-trained SDN is refined on KITTI dataset, do you just use similar sparse depth maps as ground truth to refine the SDN? And this means only about 3%-4% depth information is actually tuned?

**Comment by rui1996** [comment]
Please check Table 1 in paper for details.
